scopes/component/ui/component-meta/component-overview.tsx

```diff
@@ -44,30 +53,59 @@ export function ComponentOverview({
     const origin = isBrowser ? window.location.origin : undefined;
     finalElementsUrl = origin && elementsUrl ? `${origin}${elementsUrl}` : undefined;
   }
+
+  type CompWithTitleBadgesProps = { position: TitleBadgePosition; children: ReactNode };
+  function ComponentWithTitleBadges({ position, children }: CompWithTitleBadgesProps) {
```

Comment:
extract outside of ComponentOverview, pass props (like titleBadges) explicitly

Reply:
done